ccdImports/conversionScripts/createAllXLS.sh

```diff
@@ -21,5 +21,6 @@ ${conversionFolder}/convertJsonToXLS.sh ${configFolder}/CCD_Probate_Legacy_Cases
 ${conversionFolder}/convertJsonToXLS.sh ${configFolder}/CCD_Probate_Legacy_Search/
 ${conversionFolder}/convertJsonToXLS.sh ${configFolder}/CCD_Probate_Will_Lodgement/
 ${conversionFolder}/convertJsonToXLS.sh ${configFolder}/CCD_Probate_Standing_Search/
+${conversionFolder}/convertJsonToXLS.sh ${configFolder}/CCD_Probate_BulkScanning_ExceptionRecord/
```

Comment:
we dont need the BulkScanning one - that SS is kept uptodate by the BS team

Reply:
okay, I just merged this. I'll make a PR to remove it and delete those files